streampark-common/src/main/scala/org/apache/streampark/common/util/Utils.scala

```diff
@@ -44,14 +53,20 @@ object Utils {
 
   def isEmpty(elem: Any): Boolean = !notEmpty(elem)
 
-  def uuid(): String = UUID.randomUUID().toString.replaceAll("-", "")
+  def required(expression: Boolean): Unit = {
+    if (!expression) {
+      throw new IllegalArgumentException
+    }
+  }
 
-  def require(requirement: Boolean, message: String): Unit = {
-    if (!requirement) {
-      throw new IllegalArgumentException(s"requirement failed: $message")
+  def required(expression: Boolean, errorMessage: Any): Unit = {
```

Comment:
How about `checkArgument` due to it throw `IllegalArgumentException` when `expression == false`?

Reply:
> How about `checkArgument` due to it throw `IllegalArgumentException` when `expression == false`?

about the method name, I prefer `required`, shorter and more concise